_lessons/maui-and-mahuika/07-installing-local-packages.md

```diff
@@ -20,29 +20,22 @@ Make sure to load one of the Python versions installed on Mahuika, e.g.
 ```
 module load Python/2.7.14-gimkl-2017a
 ```
-as this will provide you with many Python modules. You can check if a module is 
-already installed by typing, for instance,
+as this will provide you with many Python modules. Our Python distributions come with many modules already installed. You can check if a Python module is installed by typing, for instance, 
 ```
-python -c "import pnumpy"
+python -c "import numpy"
```

Comment:
Why the change to `numpy` - I thought we were using `pnumpy` as an examples?

Reply:
It seems pnumpy is still the example for installing yourself, so that's OK.  

I will be taking out any mention of the sbatch option --exclusive.